.gitignore

```diff
@@ -63,11 +63,8 @@ ansible/roles/nginx/files/*cert.pem
 
 # .zip files must be explicited whitelisted
 *.zip
-!tests/dat/actions/blackbox.zip
```

Comment:
because these dont exist in this repo

Reply:
Is there an emoji for “Hey I just left it there because you guys always knock me for changing too many things at once?”. 😛